### ISSUE TITLE
openai.error.InvalidRequestError: The model `gpt-4` does not exist or you do not have access to it.

### ISSUE THREAD

**Issue by dserodio** [gnachman/gptline#1]
When I try to use gptline I get the following error:

```
Traceback (most recent call last):
  File "/Users/daniel.serodio/.local/bin/gptline", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/src/main.py", line 160, in main
    chat = create_chat_with_spinner(sanitized, temperature, functions)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/src/chat.py", line 79, in create_chat_with_spinner
    return create_chat(messages, temperature, functions, True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/src/chat.py", line 97, in create_chat
    return spin(create_chat_model)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/src/spin.py", line 34, in spin
    result = f()
             ^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/src/chat.py", line 91, in create_chat_model
    return openai.ChatCompletion.create(**args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/openai/api_resources/chat_completion.py", line 25, in create
    return super().create(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/openai/api_resources/abstract/engine_api_resource.py", line 153, in create
    response, _, api_key = requestor.request(
                           ^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/openai/api_requestor.py", line 298, in request
    resp, got_stream = self._interpret_response(result, stream)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/openai/api_requestor.py", line 700, in _interpret_response
    self._interpret_response_line(
  File "/Users/daniel.serodio/.local/pipx/venvs/gptline/lib/python3.11/site-packages/openai/api_requestor.py", line 765, in _interpret_response_line
    raise self.handle_error_response(
openai.error.InvalidRequestError: The model `gpt-4` does not exist or you do not have access to it. Learn more: https://help.openai.com/en/articles/7102672-how-can-i-access-gpt-4.
```

This is very weird, because from glancing at the code it seems like `gpt-3.5-turbo` [is the default](https://github.com/gnachman/gptline/blob/6805e77141a84896df6582f544bffd16bf099020/src/main.py#L182), and I didn't find any occurrences of `gpt-4`

I can't use GPT-4 because I'm not a ChatGPT Plus subscriber.

Thanks!

**Comment by gnachman** [comment]
I just published 1.0.7 which actually defaults to 3.5-turbo. Upgrade and try it again

**Comment by dserodio** [comment]
Thanks, that's why the behavior didn't match the code I saw I was running 1.0.3 and looking at commit 6805e77 :)

I had also messed up my `pipx` command: I had tried to update gptline before reporting this issue but used `pipx install --force gptline` instead of `pipx upgrade gptline`

Working fine with 1.0.7.

Thanks for the prompt reply 🙇🏼
